app/controllers/api/v2x0/taggings_controller.rb

```diff
@@ -0,0 +1,38 @@
+module Api
+  module V2x0
+    class TaggingsController < ApplicationController
+      include Api::V1::Mixins::IndexMixin
+
+      # Present these as tags
+      private_class_method def self.api_doc_definition
+        @api_doc_definition ||= api_doc.definitions["Tag"]
+      end
+
+      def self.presentation_name
+        "Tag".freeze
+      end
+
+      def create
+        primary_instance = primary_collection_model.find(request_path_parts["primary_collection_id"])
+        tag = Tag.find_or_create_by!(Tag.parse(params_for_create["tag"]))
+        primary_instance.tags << tag
```

Comment:
Found when adding functionality to Catalog (https://github.com/RedHatInsights/catalog-api/pull/495):
This fails if the tag already exists on `primary_instance`, I just added a guard clause and it went away.

Reply:
I wonder if we should return a `409 Conflict` instead?

Reply:
@lindgrenj6 I wound up returning a 304 here if it is already tagged

Reply:
:+1: sounds good to me! I'll get that added to my implementation over on Catalog.